scripts/import/import_clinvar_xml

```diff
@@ -766,6 +806,28 @@ sub update_variation_set{
 
 }
 
+=head2 update_variation_feature
+- update variation_feature evidence and variation_set_id
+=cut
+
+sub update_variation_feature{
+  my $vf      = shift;
+
+  my $vf_upd_sth = $dbh->prepare(qq[
+      UPDATE variation_feature
+      SET
+        variation_set_id = CONCAT_WS(",", variation_set_id, ?),
+        evidence_attribs = CONCAT_WS(",", evidence_attribs, '$pheno_evidence_id')
+      WHERE variation_feature_id = ? ] );
+  my $new_sets = "$pheno_set_id,$clinvar_set_id";
+  if ($omimVarSet{$vf->variation->dbID}) {
+    $new_sets .=",$omim_set_id";
+  }
+  $vf_upd_sth->execute( $new_sets,  $vf->dbID );
```

Comment:
Which db server is this working on? I tried it on prod4 and got:
 DBD::mysql::st execute failed: Data truncated for column 'variation_set_id' at row 1 at scripts/import/import_clinvar_xml line 826.

It looks like '23,30,24' is being treated as a string.

Reply:
Thanks for checking. There is a setup difference between prod4 and the other servers which leads to this error and not on the others, I created a jira ticked for it ENSVAR-2979.